packages/app/src/systems/CRX/background/services/BackgroundService.ts

```diff
@@ -134,7 +134,7 @@ export class BackgroundService {
     const origin = serverParams.origin;
 
     if (origin) {
-      await ConnectionService.removeConnection(origin);
+      await ConnectionService.removeConnection({ origin });
```

Comment:
In fact, the pattern is having an input type to be able to just use this type across all things, like machines and others places to have the same input across functions. Was because of that I changed because I need to use this service in other places and I want it to use the same `({ input })` pattern.

Reply:
Should we change the methods to follow the pattern then?